Resources/Prototypes/Entities/Objects/tape_recorder.yml

```diff
@@ -0,0 +1,223 @@
+- type: entity
+  name: tape recorder
+  parent: BaseItem
+  description: Anything said into this device can and will be used against you in a court of space law.
+  id: TapeRecorder
+  components:
+    - type: Sprite
+      sprite: Objects/Devices/tape_recorder.rsi
+      netsync: false
+      layers:
+        - state: taperecorder_empty
+          map: ["base"]
+    - type: Item
+      sprite: Objects/Devices/tape_recorder.rsi
+      heldPrefix: taperecorder_empty
```

Comment:
it works not like that
.rsi should contain states `inhand_left` and `inhand_right`, they will be used by default
if you write `heldPrefix: bill` it will use `bill_inhand_left` and `bill_inhand_right`
so actually you can remove that line and just make states with default names
and actually you havent inhand states :trollface: